src/znc.cpp

```diff
@@ -2121,3 +2152,10 @@ void CZNC::LeakConnectQueueTimer(CConnectQueueTimer* pTimer) {
 }
 
 bool CZNC::WaitForChildLock() { return m_pLockFile && m_pLockFile->ExLock(); }
+
+void CZNC::DisableConfigTimer() {
+  if (m_pConfigTimer) {
+    m_pConfigTimer->Stop();
+    m_pConfigTimer = nullptr;
```

Comment:
I've had a closer look at Csocket, it looks you're right... `Pause()` could also help, but not very well in this case either. While I don't like this approach too much, it's better than some alternatives.

But please reformat the code. ZNC doesn't use 2 spaces for indentation.